docs/testing/workspaces-migration-guide.md

```diff
@@ -0,0 +1,319 @@
+
+Unlike NEAR Simulator, `workspaces-rs` uses an actual NEAR node and makes all calls through it. First, you need to decide which network you want your tests to be run on:
+* `sandbox` - perfect choice if you are just interested in local development and testing; `workspaces-rs` will instantiate a [sandbox](https://github.com/near/sandbox) instance on your local machine which will run an isolated NEAR node.
+* `testnet` - an environment much closer to the real world; you can test integrations with other deployed contracts on testnet without bearing any financial risk.
+* `mainnet` - a network with reduced amount of features due to how dangerous it can be to do transactions there, but can still be useful for automating deployments and pulling deployed contracts.
+
+In this guide we will be focusing on `sandbox` since it covers the same use cases NEAR Simulator did. But of course feel free to explore whether other networks can be of potential use to you when writing new tests/workflows.
+
+One of the ways to initialize simulator and deploy a contract is shown below (the other way is through `deploy!` macro which we will look at in the next section):
+
+```rust title="Deployment - near-sdk-sim"
+use near_sdk_sim::{init_simulator, to_yocto};
+
+near_sdk_sim::lazy_static_include::lazy_static_include_bytes! {
+    WASM_BYTES => "res/contract.wasm",
+}
+
+const ID: &str = "contract-id";
+
+...
+
+let root = init_simulator(...);
+let contract = root.deploy(&WASM_BYTES, ID.parse().unwrap(), to_yocto("5"));
+```
+
+Although `workspaces-rs` provides a way to specify the account id for a contract to be deployed, usually it does not matter in the context of a single test. If you are fine with generating a random developer account and initializing it with 100N, then you can use replace the snippet above with this:
+
+```rust title="Deployment - workspaces-rs"
+let worker = workspaces::sandbox().await?;
+let contract = worker.dev_deploy(include_bytes!("../res/contract.wasm")).await?;
+```
+
+Alternatively, use this if you care about the account id:
+
+```rust title="Deployment - workspaces-rs (with explicit account id)"
+let worker = workspaces::sandbox().await?;
+let (_, sk) = worker.dev_generate().await;
+let id: AccountId = "contract-id".parse()?;
+let contract = worker
+    .create_tla_and_deploy(
+        id,
+        sk,
+        include_bytes!("../examples/res/non_fungible_token.wasm"),
+    )
+    .await?
+    .result;
+```
+
+Or, if you want to create a subaccount with a certain balance:
+
+```rust title="Deployment - workspaces-rs (with explicit balance)"
+use near_units::parse_near;
+
+let worker = workspaces::sandbox().await?;
+let id: AccountId = "contract-id".parse()?;
+let contract = worker
+    .root_account()
+    .create_subaccount(&worker, id)
+    .initial_balance(parse_near!("5 N"))
+    .transact()
+    .await?
+    .result;
+```
+
+:::danger
+TODO: Is there a reason why we can't control the initial balance with `dev_deploy`?
```

Comment:
@ChaoticTempest question to you

Reply:
`dev_deploy` can't supply the initial balance since testnet controls this amount in the helper contract which is what we're using to create dev accounts in testnet. So, to make it simple, we don't supply it at all for sandbox even. If someone wants to supply an amount in sandbox, they can grab the root account and do:
```rust
let root = worker.root_acount();
root.create_subaccount(...)
   .initial_balance(...)
   ...
```